src/m365/spo/commands/file/file-sharinginfo-get.ts

```diff
@@ -174,13 +175,14 @@ class SpoFileSharingInfoGetCommand extends SpoCommand {
     let requestUrl: string = '';
 
     if (args.options.fileId) {
-      requestUrl = `${args.options.webUrl}/_api/web/GetFileById('${escape(args.options.fileId as string)}')/?$select=ListItemAllFields/Id,ListItemAllFields/ParentList/Title&$expand=ListItemAllFields/ParentList`;
+      requestUrl = `${args.options.webUrl}/_api/web/GetFileById('${args.options.fileId}')/?$select=ListItemAllFields/Id,ListItemAllFields/ParentList/Title&$expand=ListItemAllFields/ParentList`;
     }
     else {
-      requestUrl = `${args.options.webUrl}/_api/web/GetFileByServerRelativePath(decodedUrl='${formatting.encodeQueryParameter(args.options.fileUrl as string)}')?$select=ListItemAllFields/Id,ListItemAllFields/ParentList/Title&$expand=ListItemAllFields/ParentList`;
+      const serverRelPath = urlUtil.getServerRelativePath(args.options.webUrl, args.options.fileUrl!);
```

Comment:
I noticed that in some places we call it `serverRelUrl` and in other `serverRelPath`. Let's choose for one name and use it consistently throughout. Also, let's consider spelling out `rel` as `relative` to make it clearer to follow for folks for whom English isn't the primary language.